Assets/TriggerExplosion.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TriggerExplosion : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject m_Destruction;
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if(other.CompareTag("Meteor"))
+        {
+            Debug.Log("METEORRRRR");
```

Comment:
This logic is a little weird. What is `TriggerExplosion` attached to? Why is it checking for a `Meteor`? In general, we should avoid specialized tags. We have a hard limit of 32 tags so it should be used for game-level stuff (Camera, Player, etc.). Something so specific would probably do better to check for the existence of a script instead.